.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Info Required**
+- Which version of the library do you actually use?
+
+- Do you have the localization permission granted?
+
+- Are you sending parameters to the activity through the bundle?
+
+- Could you describe what are the actions do you make to raise that error?
+
+- Android monitor shows any log related to the library when the error is shown?
+
+- Any other thing that could help me to reproduce the error?
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**

```

Comment:
How are we going to use this info? I don't see why we want this. Same with the **Smartphone** section below.

Also, maybe would be nice to ask for the Google Play Services version too. It can be useful.